lib/fastly_nsq/rake_task.rb

```diff
@@ -37,19 +37,16 @@ def run_tasks
       listener_threads = []
 
       topics.each do |topic|
-        listener_threads << Thread.new do
+        thread = Thread.new do
           wrap_helpful_output(topic) do
             MessageQueue::Listener.new(topic: topic, channel: channel).go
           end
         end
+        thread.abort_on_exception = true
```

Comment:
I _think_ I like this better than the loop (`unless status` made my brain a little hurty), but I'm curious: why not `Thread.abort_on_exception` ?